.github/workflows/ci-llama.yaml

```diff
@@ -9,6 +9,10 @@ name: Llama Benchmarking Tests
 on:
   workflow_dispatch:
   pull_request:
+    paths:
+      - '.github/workflows/ci-llama.yaml'
+      - 'sharktank/**'
+      - '*requirements*.txt'
   schedule:
```

Comment:
I left a comment on https://github.com/nod-ai/SHARK-Platform/pull/363#discussion_r1822842474. I'd rather see this run on _either_ `schedule` _or_ `pull_request, push`.

The workflow name also isn't consistent with others (`ci-llama` but `Llama Benchmarking Tests` name doesn't sort along with the others at https://github.com/nod-ai/SHARK-Platform/actions/workflows/ci-llama.yaml)

Reply:
Here's what I mean with sorting: 
![image](https://github.com/user-attachments/assets/0cb2fc80-e534-48a4-be02-5b155120ca53)

Reply:
We're going to have more models to test and benchmark, so I'd like us to get better organized about how we define workflows.